### PR TITLE
arx-libertatis: migrate from homebrew-games

### PR DESCRIPTION
**Pull request by mistydemeo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Extracted from #9753. A couple patches are needed to fix stable; going to also contact them asking for a new release.

### CODE REVIEW

File: Formula/arx-libertatis.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    # Install prebuilt icons to avoid inkscape and imagemagick deps
+    if build.head?
+      (buildpath/"arx-libertatis-data").install resource("arx-libertatis-data")
+      args << "-DDATA_FILES=#{buildpath}/arx-libertatis-data"
+    end
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
```
Comment (ilovezfs) [inline Formula/arx-libertatis.rb]: does just make install die?

File: Formula/arx-libertatis.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  end
+
+  option "without-innoextract", "Build without arx-install-data"
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :build
+  depends_on "glm" => :build
+  depends_on "freetype"
+  depends_on "glew"
+  depends_on "innoextract" => :recommended
```
Comment (ilovezfs) [inline Formula/arx-libertatis.rb]: should be below sdl

File: Formula/arx-libertatis.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  depends_on "sdl"
+
+  def install
+    args = std_cmake_args
+
+    # The patches for these aren't straightforward to backport because of
+    # other changes; these minimal inreplaces get it building.
+    # HEAD is fine, and the next stable release will contain these changes.
```
Comment (ilovezfs) [inline Formula/arx-libertatis.rb]: should probably include the link to the corresponding commit

**Comment by ilovezfs** [comment]
Looks like some extra special joy on Yosemite :)

File: Formula/arx-libertatis.rb (hunk 1 above)
Comment (mistydemeo) [inline Formula/arx-libertatis.rb]: No longer necessary, it looks like. Removed the `make`.

File: Formula/arx-libertatis.rb (hunk 2 above)
Comment (mistydemeo) [inline Formula/arx-libertatis.rb]: Moved.

**Comment by ilovezfs** [comment]
Still dies later on
```
/tmp/arx-libertatis-20170212-70819-2msvmp/arx-libertatis-1.1.2/src/platform/Time.cpp:92:10: fatal error: 'mach/clockid_types.h' file not found
#include <mach/clockid_types.h>
         ^
1 error generated.
make[2]: *** [CMakeFiles/common.dir/src/platform/Time.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/common.dir/all] Error 2
make: *** [all] Error 2
/Users/brew/foo/brew/Library/Homebrew/debrew.rb:11:in `raise'
BuildError: Failed executing: make install
```

**Comment by mistydemeo** [comment]
Seems to have built fine on the bots?

**Comment by ilovezfs** [comment]
I had to comment out that include on Yosemite. Building in non-standard prefix.

**Comment by mistydemeo** [comment]
It looks like it's a system header it's trying to find - wonder why prefix would affect it.

**Comment by ilovezfs** [comment]
Just hit on the bot too: https://bot.brew.sh/job/Homebrew%20Core/16295/version=yosemite/console

**Comment by ilovezfs** [comment]
it's the inreplace hehe

**Comment by mistydemeo** [comment]
Inreplace fixed 😅

File: Formula/arx-libertatis.rb (hunk 3 above)
Comment (ilovezfs) [inline Formula/arx-libertatis.rb]: I think the corresponding commit is https://github.com/arx/ArxLibertatis/commit/39fb9a0e3a6888a6a5f040e39896e88750c89065

Comment (ilovezfs) [inline Formula/arx-libertatis.rb]: ah I see you added it farther down! thx

**Review by ilovezfs — APPROVED** [review]

**Pull request closed without merging**